packages/store/src/decorators/select/select.ts

```diff
@@ -7,20 +7,13 @@ import { createSelectObservable, createSelectorFn } from './symbols';
 export function Select(rawSelector?: any, ...paths: string[]): PropertyDecorator {
   return function(target: any, propertyKey: string | symbol): void {
     const name: string = propertyKey.toString();
-    const selectorId = `__${name}__selector`;
+    const selectorId: unique symbol = Symbol(`__${name}__selector`);
```

Comment:
`Symbol` is not used anywhere else in the library. Adding this will eliminate support for IE and there is no pollyfill that can be used.